src/components/form/Form.stories.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import MemberFields from './MemberFields';
+
+storiesOf('Body', module)
+  .add('default', () => (
+    <MemberFields />
```

Comment:
no idea what's this for, but looks cool. @silviopaganini maybe the other fieldset for advertisers should be in storybook as well? how is that used? maybe the form with the radio button as well? or the member's card switching to form? 😂

Reply:
That’s just to quickly debug UI, rendering it separately from the rest of the build. When we need test/develop the rest we’ll create them